src/test/java/com/google/javascript/clutz/partial/declare_legacy_namespace.js

```diff
@@ -0,0 +1,7 @@
+
+goog.module('declare.legacy.namespace');
+goog.module.declareLegacyNamespace();
+
+class Class {}
```

Comment:
Because you are using `visitKnownTypeValueAlias` without any guarantees that the symbols are really are of the type-and-value variety (classes, enums), I don't expect this code to work correctly with interfaces. That said, lets leave it for a follow-up PR, and add a TODO (or even add interface in this test and a comment in the output "this is wrong, will fix later").

Reply:
Added a comment.